tests/output_melt_track/solution/solution-00000.0000.gnuplot

```diff
@@ -0,0 +1,4487 @@
+# This file was generated by the deal.II library.
```

Comment:
Remove this file. We are not interested in the solution, only in the particle properties